### PR TITLE
Add Sourcegraph Social

### PR DESCRIPTION
**Pull request by pecigonzalo**
Initial handbook page to share our social calendar.

### CODE REVIEW

File: company/remote/calendar.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Social Calendar
```
Comment (sqs) [inline company/remote/calendar.md:1]:
```suggestion
# Social calendar
```

https://about.sourcegraph.com/handbook/communication/style_guide#sentence-case

File: company/remote/index.md [hunk 2]
```diff
@@ -53,6 +53,7 @@ After that it is just a matter of not checking your work email (or Slack) at tim
 
 - [Our journey to all-remote](https://about.sourcegraph.com/blog/our-journey-to-all-remote)
 - [Tips for working remotely](tips.md)
+- [Sourcegraph Social Calendar](calendar.md)
```
Comment (sqs) [inline company/remote/index.md:56]:
```suggestion
- [Social calendar](calendar.md)
```

https://about.sourcegraph.com/handbook/communication/style_guide#sentence-case

File: company/remote/calendar.md (hunk 1 above)
Comment (sqs) [inline company/remote/calendar.md:1]: IMO should rename file to `social_calendar.md`

File: handbook/communication/index.md [hunk 3]
```diff
@@ -35,6 +35,7 @@ The following places are not sources of truth. Treat documents and conversations
 ## Meetings
 
 - [Company meeting](company_meeting.md) (Mondays 10:30-11:00am PST/PDT)
+- [Social meetings](../../company/remote/calendar.md)
```
Comment (sqs) [inline handbook/communication/index.md:38]:
```suggestion
- [Social calendar](../../company/remote/calendar.md) meetings
```

to make it easy to scan this and see what it's about

**Review by sqs — APPROVED** [review]

File: company/remote/index.md (hunk 2 above)
Comment (pecigonzalo) [reply company/remote/index.md:56]: I have changed the case in all of them now, but I chose to use that casing as it is the calendar name and not just a calendar that is called Social.

**Pull request merged**